### PR TITLE
[full-ci][tests-only]Bump ocis commit id for tests

### PR DESCRIPTION
**Pull request by amrita-shrestha**
This PR bumps ocis commit id for tests
Part of: https://github.com/owncloud/QA/issues/806
https://github.com/owncloud/web/issues/8929

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oC10</strong> https://drone.owncloud.com/owncloud/web/35229/11/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35229/tracing/public-link-alice-2023-4-27-07-36-22.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35229/tracing/re-sharing-alice-2023-4-27-07-38-41.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35229/tracing/re-sharing-brian-2023-4-27-07-38-28.zip 

#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/35229/tracing/re-sharing-carol-2023-4-27-07-38-34.zip 


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/35229/65/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingPublicBasic-publicLinkCreate_feature-L165.png
  


</p></details>

File: tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature [hunk 1]
```diff
@@ -172,7 +172,7 @@ Feature: Create public link shares
       | uid_owner   | Alice          |
       | permissions | read           |
       | path        | /simple-folder |
-      | name        | Link           |
+      | name        | Quicklink      |
```
Comment (saw-jan) [inline tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature:175]: might not need it: https://github.com/owncloud/web/issues/8929#issuecomment-1525030185

**Comment by JammingBen** [comment]
This PR probably needs a rebase because https://github.com/owncloud/web/pull/8934 has been merged.

**Review by saw-jan — APPROVED** [review]

**Pull request merged**